pkg/rbd/nodeserver.go

```diff
@@ -303,15 +303,37 @@ func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeS
 	// Publish Path
 	fsType := req.GetVolumeCapability().GetMount().GetFsType()
 	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
+	source := devicePath
+	readOnly := false
+	existingFormat, err := diskMounter.GetDiskFormat(source)
+	if err != nil {
+		return err
+	}
```

Comment:
Move `if fsType == "" { fsType = "ext4" } here.

Reply:
not required, as the sidecar fills `ext4` as the default FS. So, removed it.

Reply:
https://github.com/ceph/ceph-csi/issues/633